addons/google_account/google_account.py

```diff
@@ -89,14 +89,17 @@ def _get_google_token_json(self, cr, uid, authorize_code, service, context=None)
 
         headers = {"content-type": "application/x-www-form-urlencoded"}
 
-        try:
-            uri = self.get_uri_oauth(a='token')
-            data = werkzeug.url_encode(params)
+        if client_id and client_secret:
```

Comment:
Alright this makes sense. It was for readability that I was thinking about the `else` statement but you're right. Thanks for the information about cyclomatic complexity too, quite a good read.